### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by thneves**
- Created user interactions with the game to play on the board

### CODE REVIEW

File: bin/.rubocop.yml
Comment (maelfosso) [inline bin/.rubocop.yml]: Sorry but that is not the good `rubycop.yml` file. This one is for Ruby on Rails project.
Kindly refer to this one https://github.com/microverseinc/linters-config/tree/master/ruby

File: bin/main
Comment (maelfosso) [inline bin/main]: Instead of two loops, kindly implement a change current user concept.

File: bin/main
Comment (maelfosso) [inline bin/main]: Something like this will be better
```rb
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for the well-done job so far.
However, there are some missing requirements

1. Kindly setup Rubocop GitHub Action. Follow this [link](https://github.com/microverseinc/linters-config/tree/master/ruby) to know how to

2. Your `lib` folder is missing. Kindly create one with `.gitkeep`, an empty file inside it.

3. Your game flow is almost good. Some points are missing. Here are them
* [x] `[OPTIONAL]` game asks players for their names
* [x] `[REQUIRED]` game tells which player turn it is (without computing who the winner is) :heavy_check_mark: 
* [x] `[REQUIRED]` game asks the player to select from available moves :heavy_check_mark: 
* [x] `[REQUIRED]` game informs player if the selected move is invalid (without computing what an invalid move is) :heavy_check_mark: 
* [x] `[REQUIRED]` game displays board after the player move (without computing which spots on the board are taken) :heavy_check_mark: 
* [x] `[REQUIRED]` game informs player if the selected move is a winning move (without computing the winning move)
* [x] `[REQUIRED]` game informs player if the selected move is a draw move (without computing draw)
* [x] `[REQUIRED]` game repeats all actions for the next player's move

Kindly, please, kind in mind that in this step you are supposed to only **mock** the basic flow of the game. This means implementing a user interface that displays prompt for users (print/puts) and collects user input (`get.chomp`). No game logic (for example checking who is a winner) is required in this milestone

Once done, please, submit a new code review request
Thanks

Mael FOSSO

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

 Good job so far, team! 👍 
However, we still need to do some changes before this project gets approved 💪 
Kindly check my comments below 👇 , implement the changes nd resubmit your project for a new code review ⏩ 
Good luck! 😺 

## Mandatory changes
- In the user-interface branch I still cannot see any of the changes required previously. I cannot see a `lib` folder, and the linters are not set up correctly. Kindly follow the instructions from the previous review to set up the linters, the folder and apply the changes in your loops. 
- You neither have license or readme file, please implement it.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are still required before this project can be approved. Let's check them out.__

**[ REQUIRED CHANGES ]**

_Please note that the changes in this category are compulsory: they must be made before your project can be approved._

- [x] **You're having a merge conflict. Kindly resolve all conflicts before requesting another review request.**



- [x] **Your GitHub Actions linter checks are not running.**
- It looks like you did not set up the linters workflow for this milestone. Please refer to [this](https://github.com/microverseinc/linters-config/tree/master/ruby) page to properly set up your linters config. 
- Please fix all linter errors too.

- [x] **You should find a way to terminate your loop at some point.**
- For now, your game still asks the player to make a move even when all the moves have been taken. You should find a way to end the game when all moves have been made.
- This might not be fully implemented in this milestone, just find a way to terminate your game after some round of moves.


**[ COMMENTS ]**

- _I strongly advise that you work on getting this milestone fixed and approved before submitting the next one. This is because the next milestone strongly depends on this one. Also, to avoid merge conflicts in the future, this is important._

- _Did you need help? Feel free to reach out to any of the previous TSEs or me. We're here to help._

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_sign: 
### HELLO :wave: VERY NICE PROGRESS. GOOD WORK MAKING THE CHANGES AS REQUESTED BY THE PREVIOUS TSE :+1: 

### KINDLY FIX THE FOLLOWING BEFORE WE PROGRESS :arrow_down: 

- [x] The game informs the player if the selected move is a winning move (without computing the winning move). The game should be able to tell us if a player wins. You don't have to write the logic for this it doesn't have to be the real winner.

- [x] The the game informs player if the selected move is a draw move (without computing draw). Just like the winning move above the same should be done for the drawing move.

**if you have any question kindly reach me on slack @peterrobert**

### HAPPY CODING :computer:

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**
